src/middleware/validate-body.js

```diff
@@ -0,0 +1,18 @@
+const { validationResult } = require("express-validator");
+
+const validateBody = (req, res, next) => {
+  const errors = validationResult(req);
+  if (!errors.isEmpty()) {
+    throw new Error(
+      JSON.stringify({
+        status: "UNPROCESSABLE_ENTITY",
```

Comment:
Usually the HTTP status code for the error goes here, not a string of the status label.  Did you mean to import the `ERROR_CODES` object from the error-handler module and refer to the labelled value there?